src/components/Guide.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Grid, Typography } from '@mui/material'
+
+export default function Guide() {
+  const items = [
+    {
+      title: 'Como funciona?',
+      content:
+        'O Fluxo Ágil oferece, aos estudantes da Universidade de Brasília, uma recomendação de quais disciplinas cursar nos próximos semestres para que você se forme no menor tempo possível seguindo a quantidade de créditos que deseja cursar.',
+    },
+    {
+      title: 'Como obter seu histórico?',
+      content:
+        'Para utilizar a plataforma, primeiramente é necessário fazer a emissão de seu histórico pelo siga no menu de ensino. Após fazer a emissão, é preciso submeter seu histórico na plataforma e será identificado todas as disciplinas que você ainda precisa fazer. Em seguida, basta escolher quantos créditos você deseja realizar nos próximos semestre e pronto! O sistema realizará a recomendação de matérias para os próximos semestres.',
```

Comment:
```suggestion
        'Para utilizar a plataforma, primeiramente é necessário fazer a emissão de seu histórico pelo SIGAA no menu de ensino. Após fazer a emissão, é preciso submeter seu histórico na plataforma e será identificado todas as disciplinas que você ainda precisa fazer. Em seguida, basta escolher quantos créditos você deseja realizar nos próximos semestre e pronto! O sistema realizará a recomendação de matérias para os próximos semestres.',
```
Seria bom se tivesse um link pro SIGAA.

---

Sugestão as instruções um pouco mais amigáveis. Exemplo: 

"Para receber a recomendação, você só precisa pegar o seu histórico acadêmico do SIGAA e (...) Escolha a quantidade de créditos que você deseja cursar por semestre e tá pronto o sorvetinho!"

Reply:
Alterei um pouco do texto.